Week-5/Solution4.test.js

```diff
@@ -0,0 +1,4 @@
+const mathOperations = require("./Solution4");
```

Comment:
Any specific reason for using `require` when we can `import` the statement using `Es6` syntax?